src/Components/TodoList.js

```diff
@@ -0,0 +1,38 @@
+// TodoList.js
+import React from 'react';
+import TodoListItem from './TodoListItem';
+import styles from './TodoList.module.css'; // Import the CSS module
+import PropTypes from 'prop-types'; // Import PropTypes
+
+function TodoList({ todoList, onRemoveTodo }) {
+    // Destructure the props todoList and onRemoveTodo from the incoming props object
+
+    return (
+        <div className={styles.TodoListContainer}>
+            <div className={styles.TodoList}>
+                {/* Render a heading */}
+                <h2 className={styles.ListHeader}>Todo List</h2>
+                <ul className={styles.List}>
+                    {/* Map over the todoList array */}
+                    {todoList.map(todo => (
+
+                        <TodoListItem
+                            key={todo.id} // Assign a unique key to each TodoListItem
+                            todo={todo} // Pass the todo object as a prop to TodoListItem
+                            onRemoveTodo={onRemoveTodo} // Pass the onRemoveTodo function as a prop to TodoListItem
+                        />
+
+                    ))}
+                </ul>
+            </div>
+        </div>
+    );
+}
+TodoList.propTypes = {
+
+    todoList: PropTypes.func, // PropTypes for the 'todoList' prop
```

Comment:
Small `PropType bug here: 

the `todoList`  actually passes an array of objects, not a function

Reply:
Actually I'm going to go ahead and approve this PR, since the only change you need to make is a small one. No need to resubmit after you clear up the proptype error =D